### PR TITLE
Feature/separate runners

### PR DESCRIPTION
**Pull request by makosblade**
This PR:
- enables flexible archive running 
- differentiates between integrations and archives
- cleans up the launchpad integration code flows to enable ^
- Adds tests with .yamls for a couple basic archives

### CODE REVIEW

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt [hunk 1]
```diff
@@ -108,9 +124,120 @@ data class Integration(
...
+        @JsonProperty(DESTINATION) val destination: String
+): Transferable {
+    @JsonIgnore
+    override fun getLaunchPad(): String {
+        return getArchiveSql()
+    }
+
+    @JsonIgnore
+    override fun getMasterTableSql(): String {
+        return ""
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt:201]: 🤔

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt [hunk 2]
```diff
@@ -108,9 +124,120 @@ data class Integration(
...
+    }
+
+    @JsonIgnore
+    override fun getMergeTableSql(): String {
+        return ""
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt:206]: 🧐

File: src/test/kotlin/com/openlattice/launchpad/LaunchpadSmokeTests.kt [hunk 3]
```diff
@@ -25,21 +24,32 @@ class LaunchpadSmokeTests {
...
 
     companion object {
-        val logger = LoggerFactory.getLogger(LaunchpadSmokeTests::class.java)
         @JvmStatic
-        fun runTestValidateAndCleanup(config: IntegrationConfiguration, vararg sortColumn: String ) {
-            IntegrationRunner.configureOrGetSparkSession( config ).use { session ->
+        fun runArchiveValidateAndCleanup(config: IntegrationConfiguration, vararg sortColumn: String ) {
+            configureOrGetSparkSession( config ).use { session ->
+                ArchiveRunner.runArchives( config, session )
+//                IntegrationValidator.validateIntegration( config, integrationPaths, *sortColumn )
```
Comment (katherinebernstein) [inline src/test/kotlin/com/openlattice/launchpad/LaunchpadSmokeTests.kt:31]: why is this piece commented out?

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt (hunk 1 above)
Comment (makosblade) [reply src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt:201]: It's not used in the archive flow at all

File: src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt (hunk 2 above)
Comment (makosblade) [reply src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationConfiguration.kt:206]: It's not used in the archive flow either

File: src/test/kotlin/com/openlattice/launchpad/LaunchpadSmokeTests.kt (hunk 3 above)
Comment (makosblade) [reply src/test/kotlin/com/openlattice/launchpad/LaunchpadSmokeTests.kt:31]: I haven't written a validator for archives yet, just the current smoke test that runs them.

**Review by katherinebernstein — APPROVED** [review]

**Pull request merged**